xtra-libp2p/src/endpoint.rs

```diff
@@ -162,8 +162,8 @@ pub enum Error {
     BadConnection(#[from] yamux::ConnectionError), // TODO(public-api): Consider removing this.
     #[error("Address {0} does not end with a peer ID")]
     NoPeerIdInAddress(Multiaddr),
-    #[error("Either currently connecting or already connected to peer {0}")]
-    AlreadyConnected(PeerId),
+    #[error("Either currently connecting to peer {0}")]
```

Comment:
`Either` ...`or` ? 

Sentence is not complete. Did you mean? 

```suggestion
    #[error("Already connecting to peer {0}")]
```